app/views/projects/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h3>Project info</h3>
+
+<p> Name: <%= @project.name %></p>
+<p> Material: <%= @project.material %></p>
+<p> Challenge Theme: <%= @challenge.theme %></p>
+<p> Number of contestants: <%= @count %></p>
```

Comment:
See my comments in your `project_controller`. 
Line 5 - you would want to just reach through `@project` to get to the challenge theme.
```ruby
<p> Challenge Theme: <%= @project.challenge.theme %></p>
```
Line 6 - you would want to build a method in your `Project` model to get the count. 
```ruby
<p> Number of contestants: <%= @project.method_to_get_count %></p>
```